packages/react-error-overlay/middleware.js

```diff
@@ -11,6 +11,11 @@
 const { launchEditor } = require('react-dev-utils/launchEditor');
 
 module.exports = function createLaunchEditorMiddleware() {
+  if (process.platform === 'win32' && !process.env.REACT_EDITOR) {
+    const { launchPowerShellAgent } = require('react-dev-utils/launchEditor');
+    launchPowerShellAgent();
```

Comment:
Or wait. Just tested it again. It seems to break the display of the information about the editor integration if `powershell.exe` is not found. I look into that :)